static/js/reducers/ui.js

```diff
@@ -35,43 +35,41 @@ import { PERSONAL_STEP } from '../constants';
 import type { Action } from '../flow/reduxTypes';
 
 export type UIState = {
-  workHistoryEdit:              boolean;
-  workDialogVisibility:         boolean;
   educationDialogVisibility:    boolean;
   educationDialogIndex:         number;
   educationDegreeLevel:         string;
   educationLevelAnswers:        {};
+  workHistoryEdit:              boolean;
+  workDialogVisibility:         boolean;
+  workHistoryAnswer:            ?boolean;
   userPageDialogVisibility:     boolean;
   showWorkDeleteDialog:         boolean;
   showEducationDeleteDialog:    boolean;
   deletionIndex:                ?number;
   dialog:                       {};
-  showWorkDeleteAllDialog:      boolean;
-  showEducationDeleteAllDialog: boolean;
   profileStep:                  string;
   workDialogIndex:              ?number;
   userMenuOpen:                 boolean;
   searchFilterVisibility:       {[s: string]: boolean};
   tosDialogVisibility:          boolean;
   emailDialogVisibility:        boolean;
   enrollDialogVisibility:       boolean;
-  enrollSelectedProgram:       ?number;
+  enrollSelectedProgram:        ?number;
 };
 
 export const INITIAL_UI_STATE: UIState = {
-  workHistoryEdit:            true,
-  workDialogVisibility:       false,
   educationDialogVisibility:  false,
   educationDialogIndex:       -1,
   educationDegreeLevel:       '',
   educationLevelAnswers:      {},
+  workHistoryEdit:            true,
+  workDialogVisibility:       false,
+  workHistoryAnswer:          undefined,
```

Comment:
can you use `null` instead of `undefined` here? using `undefined` makes the `workHistoryAnswer` key not show up in the dev tools until we dispatch an action that changes the value.